contracts/network/BancorNetwork.sol

```diff
@@ -167,10 +167,20 @@ contract BancorNetwork is IBancorNetwork, Upgradeable, ReentrancyGuardUpgradeabl
     );
 
     /**
-     * @dev triggered when a new pool is added
+     * @dev triggered when a pool is created
+     */
+    event PoolCreated(Token indexed pool, IPoolCollection indexed poolCollection);
```

Comment:
I think we don’t even need to include the pool collection here since the association is available in the PoolAdded event. WDYT?

Reply:
Let's keep it in symmetry with the rest of the `PoolXXX` events. We don't know how they are going to be used and if in tandem or not (or not at all)

Reply:
sounds good